quanta-t41/config.json

```diff
@@ -35,7 +35,6 @@
     },
     "workflowRoot": "workflows",
     "taskRoot": "tasks",
-    "httpTemplateRoot": "templates",
-    "httpStaticRoot": "static"
```

Comment:
Attempting to insert this skupack without that folder created by the user should fail. Is the README.md unclear about placing the required files in static?